public/app/plugins/panel/timeseries/migrations.ts

```diff
@@ -711,7 +711,7 @@ function migrateHideFrom(panel: {
   }
 }
 
-function getLegendHideFromOverride(reducer: ReducerID.allIsZero | ReducerID.allIsNull) {
+const getLegendHideFromOverride = (reducer: ReducerID.allIsZero | ReducerID.allIsNull) => {
```

Comment:
6 vs 12/dozen... but why use const for simple functions?  There are cases where setting implementation is nice, but in general I vote we keep functions functions when they are just functions

Reply:
Hm, personal preference in general, I don't have strong opinions about them, I can switch back if that works better for everybody..

Reply:
This is an interesting [quick take on it](https://dev.to/skinnypetethegiraffe/function-or-const-which-do-you-prefer-typescriptjavascript-apa). I agree slightly on readability of knowing quickly what is a function vs variable but it seems like a very minor thing. I'm ok with letting devs do whatever fits their personal preference 🤷 

maybe though that means leaving already made `function` as functions and visa versa? 😬